cypress/component/OIDCConnector/OIDCUserManagerErrorBoundary.cy.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import OIDCUserManagerErrorBoundary, { SESSION_NOT_ACTIVE, TOKEN_NOT_ACTIVE } from '../../../src/auth/OIDCConnector/OIDCUserManagerErrorBoundary';
+import { UserManager } from 'oidc-client-ts';
+import ErrorBoundary from '../../../src/components/ErrorComponents/ErrorBoundary';
+import { IntlProvider } from 'react-intl';
+
+const ThrowAbleComponent = ({ error }: { error: any }) => {
+  throw error;
+};
+
+describe('OIDCUserManagerErrorBoundary', () => {
+  it('should render children if no error is thrown', () => {
+    const fakeManager = new UserManager({
+      authority: '',
+      client_id: '',
+      redirect_uri: '',
+    });
+    cy.mount(
+      <OIDCUserManagerErrorBoundary userManager={fakeManager}>
+        <h1>Child</h1>
+      </OIDCUserManagerErrorBoundary>
+    );
+
+    cy.get('h1').should('exist');
+  });
+
+  it('Should bubble unrelated OIDC timeout error to parent error boundary', () => {
+    const fakeManager = new UserManager({
```

Comment:
@Hyperkid123 some of this test setup is duplicated between our specs. Can we move it into a `beforeEach`?